src/actions/index.ts

```diff
@@ -99,14 +99,19 @@ export const hideModal: ActionCreator<HideModalAction> = () => ({
   type: HIDE_MODAL
 });
 
-async function fetchMemberByMid(dispatch: Dispatch<AppState>, username: string) {
+async function fetchMemberByMid(
```

Comment:
P2 `fetchMemberByUsername`

Reply:
still a bunch of `ByMid` type functions left, let's make this a separate PR.